ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/commands/tracker/CommandTracker.scala

```diff
@@ -328,9 +328,7 @@ private[commands] class CommandTracker[Context](
             )
           }
           .getOrElse {
-            logger.trace(
-              "Ignoring a completion with an empty submission ID for a submission from the CommandSubmissionService."
-            )
+            logger.trace("Ignoring a completion with an empty submission ID.")
```

Comment:
Please let me know if you think it would be useful to make this a warning.

Reply:
Wouldn't this be a bug? If so, I would log it with an error even

Reply:
Currently, it would be a bug. I wanted to make it consistent with: https://github.com/digital-asset/daml/pull/11593/files#diff-544c31dcb850ebc3b7ef8b56cd349f37e4c1a9f3d1dcbc78d02834311a6a0de1R52
Changing to error then.

Reply:
However, if there is no tracked command for completion, it doesn't break anything.

Reply:
Ok, if it doesn't break anything, warn is better